projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/profile-tag.e2e-spec.ts

```diff
@@ -21,6 +21,57 @@ describe('Profile-tag component', () => {
 
     profileTagHelper.triggerLoaded();
   });
+  it('should send a CartChanged event on adding an item to cart', () => {
+    goToProductPage();
+    cy.get('cx-add-to-cart button.btn-primary').click();
+    cy.get('cx-added-to-cart-dialog .btn-primary');
```

Comment:
same case as below, maybe the cart was not updated, and therefore the cart event was not updated fast enough as cypress is too fast. 

Having a wait on the request to know when it successfully loaded will fix this issue